typescript/package.json

```diff
@@ -1,4 +1,32 @@
 {
+  "name": "kurtosis-js-lib",
+  "version": "1.0.35",
```

Comment:
Will update this version, just wanted to get an approval on the current implementation